.github/workflows/ci.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       - run: cargo check --workspace
       # Check vello (the default crate) without the features used by `with_winit` for debugging
       - run: cargo check 
+      # Check vello (the default crate) without wgpu
+      - run: cargo check --no-default-features
```

Comment:
How long does this check take?

Reply:
The CI run for that step (on this PR) was ~3 seconds.

Reply:
Fair enough! Useful to have it documented here though. I've got no objection to adding this check on that metric then.